### DISCUSSION TITLE
اسکن ای پی ورژن 6 در نسخه پایتون

### DISCUSSION THREAD

**Discussion by IRANWEBFA** [MortezaBashsiz/CFScanner#562] [Q&A]
سلام
تو نسخه پایتون میخوام رنج ای پی ورژن 6 اسکن کنم 
مشکل اینجاست ای پی رنج هایی که برای ورژن 6 قرار داده شده آی پی های بسیار زیادی رو در بر میگیره و اسکریپت تو مرحله استارت میمونه
ممنون میشم راه حلی براش بزارید

**Comment by tempookian** [comment]
سلام. دو راه به ذهنم می‌رسه که می‌تونید امتحان کنید:

1. استفاده از فلگ -r و اسکن تصادفی تعداد یا نسبت مشخص از هر سابنت
2. محدودکردن ساب‌نت در ورودی:

یعنی مثلا:

```
Number of ips in "2a06:98c1:3120::/46": 4835703278458516698824704
Number of ips in "2a06:98c1:3120::/118": 1024
```

**Reply by IRANWEBFA** [reply]
با فلگ -r هم تغییری نکرد بازم استارت نمیخوره Kill میشه
با محدود کردن ساب نت کار خیلی سخت میشه چیزی که بدست اوردم اینه تبدیل به رنج های کوچیک کرد با سایت زیر :
https://subnettingpractice.com/ipv6-subnet-calculator.html

**Reply by tempookian** [reply]
ممنون از اطلاع‌رسانی. منطق رو کلا عوض کردم که کرش نکنه. نسخه آزمایشیه. اگر تست کنید و اطلاع بدید ممنون می‌شم
```bash
python3 -m pip install --pre --force-reinstall cfscanner
```

**Comment by sirvan1133** [comment]
V6 جوابه ؟

**Comment by boyluti** [comment]
سلام  اسکنر کلاد فلر  ویندوز 2 روز کار نمیکنه. همش ارور و ای پی ک قبلا هم اسکن کردم و سالم بهش میدم پیدا نمیکنه. میشه چک کنین.  نسخه ویندوز رو اپدیت کنین با تشکر

**Reply by boyluti** [reply]
اخرین نسخه هم استفاده میکنم   1.28530 اما مشکل داره الان

**Reply by tempookian** [reply]
سلام لطفا ایشوی جدید باز کنید تا بررسی بشه. مرتبط به این بحث نیست
